frontend/test/pull-card-demo.tsx

```diff
@@ -75,4 +77,8 @@ function Row({ title, pullDatas }: { title: string; pullDatas: PullData[] }) {
   );
 }
 
+Row.defaultProps = {
```

Comment:
Hmm what's this? I wasn't aware react would examine properties on the function instance of a function component.

TIL! Looks like it used to be a thing but was deprecated in 2019:
https://github.com/facebook/react/pull/16210